app/models/manageiq/providers/redhat/inventory/persister/infra_manager.rb

```diff
@@ -14,7 +14,9 @@ def initialize_inventory_collections
     add_collection(infra, :host_storages)
     add_collection(infra, :host_switches)
     add_collection(infra, :host_virtual_switches)
-    add_collection(infra, :lans)
+    add_collection(infra, :lans) do |builder|
+      builder.add_properties(:complete => !targeted?)
```

Comment:
Note to future us, this shouldn't be needed if we don't collect lans as part of host targeted refresh